addons/point_of_sale/static/src/app/store/order_change_receipt_template.xml

```diff
@@ -26,8 +26,8 @@
 
             <!-- Receipt Body -->
             <div class="pos-receipt-body pb-5">
-                <t t-set="hasDataChanges" t-value="data.changes.data or data.changes.groupedData" />
-                <div t-if="hasDataChanges?.length" class="new-changes w-100">
+                <t t-set="hasDataChanges" t-value="data.changes.data.length or data.changes.groupedData or data.changes.title" />
```

Comment:
Are you sure that these variable are always defined ? If not maybe we should use question mark